cmd/aggregator.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+
+	"github.com/go-gota/gota/dataframe"
+	"github.com/spf13/cobra"
+
+	"github.com/redhatinsights/ros-ocp-backend/internal/processor"
+)
+
+var (
+	outputDir     string
+	aggregatorCmd = &cobra.Command{
+		Use:   "aggregator [input csv file path]",
+		Short: "aggregates CSV data",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			input_file := args[0]
+			if _, err := os.Stat(input_file); os.IsNotExist(err) {
+				fmt.Printf("CSV file: %s does not exist\n", input_file)
+				os.Exit(1)
+			}
+			if outputDir != "" {
+				if _, err := os.Stat(outputDir); os.IsNotExist(err) {
+					if err := os.MkdirAll("a/b/c/d", os.ModePerm); err != nil {
+						panic(err.Error())
+					}
+				}
+			} else {
+				outputDir, _ = os.Getwd()
+			}
+			f, err := os.Open(input_file)
+			if err != nil {
```

Comment:
Can we extract this if block which is repetitive as function and call?

Reply:
No, we can't. This is how exception handling done in idiomatic Go style.